### DISCUSSION TITLE
Is there a way to center the handles on a custom node?

### DISCUSSION THREAD

**Discussion by amir-alleyne** [xyflow/xyflow#3222] [Q&A]
I want to be able to overlay the node over the handle such that the edge gives the illusion that is coming from within the node. Is there a way to do this using ReactFlow? Thank you in advance!

**Comment by bcakmakoglu** [comment]
You can overwrite the handle styles, either with css or just passing style to a handle, something like this should work

```tsx
 <Handle
        type="source"
        position={Position.Right}
        id="a"
        style={{ top: '50%', left: '50%' }}
        isConnectable={isConnectable}
/>

**Reply by amir-alleyne** [reply]
Thank you!
